app/model/Exceptions.scala

```diff
@@ -108,6 +108,13 @@ object Exceptions {
 
   case class TooManySessionIdsException(m: String) extends Exception(m)
 
+  case class SchemeSpecificAnswerNotFound(m: String) extends Exception(m)
```

Comment:
frontend classes with the same names do not have `m` parameter. Is that expected?